docs/contributing/documentation_guidelines.md

```diff
@@ -200,9 +200,7 @@ The reference can be linked to, with an optional label, using the Markdown link
 -   [label for section](my_awesome_section)
 ```
 
-<details>
-
-<summary>Rendered output</summary>
+Rendered output:
 
 (my_awesome_section)=
 
```

Comment:
Writing a heading generates a `<section>` element that is closed just before the next heading. If we put a heading inside a `<details>` element, the `</details>` closing tag will be rendered _before_ the `</section>` closing tag (as the latter is deferred until the next heading shows up).

I don't know if this is a Sphinx limitation or a MyST limitation.

Browsers can render it just fine, because they are built to handle so many edge cases and incorrect HTML documents. However, this would trip up the ePub builder which requires valid XHTML.

For now, I removed the `<details>` element for this specific example.